components/admin/lessons/AdminLessonInputs/AdminLessonInputs.tsx

```diff
@@ -60,48 +68,42 @@ const initValues = (
   }
 ]
 
-const AdminModuleInputs = ({
+const AdminLessonInputs = <MainItem extends Item>({
   title,
   lessonId,
-  onAddModule,
-  module,
-  refetch
-}: Props) => {
+  item,
+  loading,
+  refetch,
+  onActionFinish,
+  action
+}: Props<MainItem>) => {
   const [formOptions, setFormOptions] = useState<any>(initValues())
```

Comment:
Remove `any`.

Reply:
This will throw an error because the [formChange](https://github.com/flacial/c0d3-app/blob/f4cd1990ea4af0e1e75eef1c26e28ca2e2617860/helpers/formChange.ts#L7) method has the `setState` value as `any`.